src/exoplanet/utils.py

```diff
@@ -47,20 +45,25 @@ def f(*args, **kwargs):
 def docs_setup():
     """Set some environment variables and ignore some warnings for the docs"""
     import logging
-    import warnings
 
+    # TODO: Remove?
+    # import warnings
     import matplotlib.pyplot as plt
 
-    # Remove when Theano is updated
-    warnings.filterwarnings("ignore", category=DeprecationWarning)
-    warnings.filterwarnings("ignore", category=FutureWarning)
-
+    # TODO: Remove? None of these warnings in the docs (did not check case studies yet)
+    # warnings.filterwarnings("ignore", category=DeprecationWarning)
+    # warnings.filterwarnings("ignore", category=FutureWarning)
     # Remove when arviz is updated
-    warnings.filterwarnings("ignore", category=UserWarning)
+    # TODO: Remove? Only warning left is from PyMC about QuadPotentialFullAdapt
+    # warnings.filterwarnings("ignore", category=UserWarning)
```

Comment:
As mentioned in comments, I did not notice any other warning than the `QuadPotentialFullAdapt` warning in the tutorials.

Maybe it's best to wait for case studies to be updated before removing those though?

Reply:
I updated the case studies and saw basically no warning other than `QuadPotentialFullAdapt`, so I think it's safe to remove these filters.